lib/relationships/relationships.ts

```diff
@@ -0,0 +1,279 @@
+import * as _ from 'lodash';
```

Comment:
What's the reason for this file being at `lib/relationships/relationships.ts` instead of `lib/relationships.ts`?

Reply:
The original idea was to add more files to the `lib/relationships` folder as we create new relationships.
Will refactor and move all relationships to `lib/contracts`, which seems consistent with how other contracts are defined.

Reply:
This file was removed